odoo/addons/base/tests/test_translate.py

```diff
@@ -652,18 +652,18 @@ def test_sync(self):
         env_fr = self.env(context={'lang': 'fr_FR'})
         env_nl = self.env(context={'lang': 'nl_NL'})
 
-        self.assertEqual(view.with_env(env_en).arch_db, archf % terms_en)
-        self.assertEqual(view.with_env(env_fr).arch_db, archf % terms_fr)
-        self.assertEqual(view.with_env(env_nl).arch_db, archf % terms_nl)
+        self.assertEqual(view.with_env(env_en).arch, archf % terms_en)
+        self.assertEqual(view.with_env(env_fr).arch, archf % terms_fr)
+        self.assertEqual(view.with_env(env_nl).arch, archf % terms_nl)
```

Comment:
The purpose of this test is to check the effect of `xml_translated` on a translated field (`arch_db`).
Replacing `arch_db` by `arch` introduces a perturbation layer that can hide or introduce bugs.
Can you revert your changes to this test?